docs/core/whats-new/dotnet-core-3-1.md

```diff
@@ -12,9 +12,9 @@ ms.date: 12/04/2019
 
 This article describes what is new in .NET Core 3.1. This release contains minor improvements to .NET Core 3.0, focusing on small, but important, fixes. The most important feature about .NET Core 3.1 is that it's a [long-term support (LTS)](#long-term-support) release.
 
-If you're using Visual Studio 2019, you must update to [Visual Studio 2019 version 16.4](https://visualstudio.microsoft.com/downloads/) to work with .NET Core 3.1 projects. For more information on what is new in Visual Studio, see the [Visual Studio Blog](https://devblogs.microsoft.com/visualstudio/tis-the-season-visual-studio-2019/).
+If you're using Visual Studio 2019, you must update to [Visual Studio 2019 version 16.4](https://visualstudio.microsoft.com/downloads/) to work with .NET Core 3.1 projects. For more information on what's new in Visual Studio, see [What's New in Visual Studio 2019 version 16.4](/visualstudio/releases/2019/release-notes#whats-new-in-visual-studio-2019-version-164).
 
-Visual Studio for Mac also supports and includes .NET Core 3.1, in the Visual Studio for Mac 8.4 Preview channel. You'll need to opt into the Preview channel to use .NET Core 3.1.
+Visual Studio for Mac also supports and includes .NET Core 3.1 in the Visual Studio for Mac 8.4 Preview channel. You'll need to opt into the Preview channel to use .NET Core 3.1.
```

Comment:
I feel that "the" is extra before "Visual Studio for Mac".
Also removing "Preview" because 8.4 is the current version, while 8.5 is the preview.

```suggestion
Visual Studio for Mac also supports and includes .NET Core 3.1 in Visual Studio for Mac 8.4.
```

Reply:
Considering that this was added when it was only available in the preview channel, it is correct. But now that 8.4 is released, this should be updated per your suggestion.